tests/pathfinding/test_rest.py

```diff
@@ -64,6 +66,11 @@ def request_path_with(**kwargs):
     response = request_path_with(max_paths=-1)
     assert response.json()['errors'] == 'Number of paths must be positive: -1'
 
+    # killen aller greenlets
```

Comment:
english comments, and we have to find a better solution for this. I can have a look.